fix: Missing app/services/issue-similarity.js causing webhook server crash

## Problem

The webhook server crashes on startup due to a missing import in `fly-github-webhooks/src/handlers/pr-check-runs.js`:

```
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/briandouglas/code/contributor.info/app/services/issue-similarity.js' 
imported from /Users/briandouglas/code/contributor.info/fly-github-webhooks/src/handlers/pr-check-runs.js
```

## Impact

- Prevents new deployments of webhook server to Fly.io
- Server crashes immediately on startup
- Blocks PR #862 (continue-review webhook migration)

## Root Cause

The file `app/services/issue-similarity.js` does not exist but is imported by the pr-check-runs handler.

## Steps to Reproduce

1. Try to start the webhook server:
   ```bash
   node fly-github-webhooks/src/server.js
   ```
2. Server crashes with ERR_MODULE_NOT_FOUND

## Solution Options

1. Create the missing `app/services/issue-similarity.js` file
2. Remove the import from `pr-check-runs.js` if no longer needed
3. Update the import path if the file was moved

## Related

- PR #862

--- a/fly-github-webhooks/src/handlers/pr-check-runs.js
+++ b/fly-github-webhooks/src/handlers/pr-check-runs.js
@@ -1,4 +1,3 @@
-import { generateIssueEmbedding } from '../../../app/services/issue-similarity.js';
 import Logger from '../utils/logger.js';
 
 /**
@@ -282,43 +281,12 @@ async function runPerformanceCheck(pr, repo, octokit, logger) {
 
 /**
  * Find similar issues using embeddings
+ * TODO: Re-enable once issue-similarity service is available in webhook context
  */
 async function findSimilarIssues(repoFullName, prTitle, prBody, supabase, logger) {
-  try {
-    // Generate embedding for the PR
-    const prEmbedding = await generateIssueEmbedding(prTitle, prBody || '');
-
-    // Query database for the repository
-    const { data: repo } = await supabase
-      .from('repositories')
-      .select('id')
-      .eq('full_name', repoFullName)
-      .single();
-
-    if (!repo) {
-      logger.info('Repository %s not found in database', repoFullName);
-      return [];
-    }
-
-    // Find similar issues using vector similarity
-    const { data: similarIssues, error } = await supabase.rpc('find_similar_issues', {
-      query_embedding: prEmbedding,
-      match_count: 5,
-      repo_id: repo.id,
-      similarity_threshold: 0.7,
-      exclude_issue_id: null,
-    });
-
-    if (error) {
-      logger.error('Error finding similar issues:', error);
-      return [];
-    }
-
-    return similarIssues || [];
-  } catch (error) {
-    logger.error('Error in findSimilarIssues:', error);
-    return [];
-  }
+  // Temporarily disabled - similarity check requires ML dependencies not available in webhook server
+  logger.info('Similarity check temporarily disabled for webhook server');
+  return [];
 }
 
 /**

